petushka1/to-do-list-review
# add-remove-fn-test

# Scope and flow

## :wrench: Implementation

- [x] setup local storage mock
- [x] install JSDOM environtment
- [x] mock inner function

## TEST

- [x] ADD FUNCTION
- check `ul` children number after a new `li` element added
- check array length after the element added
- check new object created
- [x] REMOVE FUNCTION
_invoked on click_
- check `ul` children number after `li` removed
- check array length after element removed
